lib/templates.js

```diff
@@ -223,7 +207,7 @@ const fCreateAPIResourceTemplate = (srv, srvDefinition, global, packageIds) => {
       let obj = {
         ordId: `${fReplaceSpecialCharacters(
           global.namespace
-        )}:apiResource:${srvName}:v1`,
```

Comment:
I would suggest that extract into getApiResourceOrdId function and follow this rule: `<ordId> := <namespace>:<conceptName>:<resourceName>:[v<majorVersion>]`